test/functional/apps/discover/_shared_links.js

```diff
@@ -52,14 +52,14 @@ bdd.describe('shared links', function describeIndexTests() {
 
   bdd.describe('shared link', function () {
     bdd.it('should show "Share a link" caption', function () {
-      var expectedCaption = 'Share a link';
+      var expectedCaption = 'Share saved';
```

Comment:
I guess this is only tested against the static part of the message?  I see the whole message as "Share saved search".  I suppose that "search" part changes to visualization or dashboard.